test/utils/utils.spec.ts

```diff
@@ -0,0 +1,49 @@
+/* tslint:disable: only-arrow-functions ter-prefer-arrow-callback */
+
+import * as chai from "chai";
+import { parseStrToNumStrict } from "../../app/utils/utils";
+
+describe("utils/utils.ts - parseStrToNumStrict", () => {
+  it("should return Nan for null", function() {
+    const parsed = parseStrToNumStrict(null);
+    chai.assert.isNaN(parsed);
+  });
+
+  it("should return Nan for undefined", function() {
+    const parsed = parseStrToNumStrict(undefined);
+    chai.assert.isNaN(parsed);
+  });
+
+  it("should return Nan for empty string", function() {
+    const parsed = parseStrToNumStrict("");
+    chai.assert.isNaN(parsed);
+  });
+
+  it("should return number", function() {
+    const parsed = parseStrToNumStrict("0");
+    chai.assert.isFinite(parsed);
+  });
+
+  it("should handle negative numbers", function() {
+    const parsed = parseStrToNumStrict("-1");
+    chai.assert.isBelow(parsed, 0);
```

Comment:
why don't you assert exact numbers in these tests? like `expect(parsed).to.be.eq(-1)`